demisto_sdk/commands/validate/tests/IF_validators_test.py

```diff
@@ -0,0 +1,332 @@
+from typing import List
+
+import pytest
+
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.tests.test_tools import create_incident_field_object
+from demisto_sdk.commands.validate.validators.IF_validators.IF100_is_valid_name_and_cli_name import (
+    IsValidNameAndCliNameValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF101_is_valid_content_field import (
+    IsValidContentFieldValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF102_is_valid_system_flag import (
+    IsValidSystemFlagValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF103_is_valid_field_type import (
+    FIELD_TYPES,
+    IsValidFieldTypeValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF104_is_valid_group_field import (
+    INCIDENT_FIELD_GROUP,
+    IsValidGroupFieldValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF105_is_cli_name_field_alphanumeric import (
+    IsCliNameFieldAlphanumericValidator,
+)
+from demisto_sdk.commands.validate.validators.IF_validators.IF106_is_cli_name_reserved_word import (
+    IsCliNameReservedWordValidator,
+)
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_msg",
+    [
+        pytest.param(
+            [create_incident_field_object(["name", "cliName"], ["case 1", "case1"])],
```

Comment:
please only pass a single content item and wrap it with `[]` in the test itself (rather than a list as param). This will ensure all cases have only one item (far-fetched, but imagine somebody adding a case with 3 items, then not changing the test, only checking the 0th failure's message)